### PR TITLE
chore(storybook, node): update storybook, node AL-3484 AL-5019

### PR DESCRIPTION
**Pull request by ianaa**
apart from storybook releated dep updates, this also includes a node update and a semantic-release update. This meant we could remove the ignore packages from dependabot config.

about the SB update:

I focused on getting resolved the issues that caused errors. That was primarily related to the stories that used slots inside of their templates. The error happened when the canvas's "show code" button was clicked on such components.  The simplest solution at this time was simply removing the canvas block, and leaving in the stories. this made fro the smallest diff in github, and in chromatic.

As a followup, I will be gradually migrating stories to `.ts` files, because officially defining stories inside `.mdx` files is no longer supported (this will be similar structure to combobox and listbox story docs).

But importantly I will be unblocked to add new components and their stories using the new recommended apis.

### CODE REVIEW

**Review by romiekos — APPROVED** [review]
I can't spot any issues. And the local build works fine for me.

File: src/stories/Button.stories.mdx [hunk 1]
```diff
@@ -818,62 +818,64 @@ Alternative pattern for non-standard icon sizes
...
+  }}>
+  {Template.bind({})}
+</Story>
 
 ## Using the label prop
 
 ```html
 <a-button label="Button"></a-button>
+<a-button label="Primary Menu">
+  Primary
```
Comment (manuatarchilogic) [inline src/stories/Button.stories.mdx:876]: was this intentional?

Comment (ianaa) [reply src/stories/Button.stories.mdx:876]: yes, I wanted to give a better example in the absence of the canvas's source code

Comment (manuatarchilogic) [reply src/stories/Button.stories.mdx:876]: I meant having a "Primary Menu" label and a "Primary" content

Comment (ianaa) [reply src/stories/Button.stories.mdx:876]: kind of? might not have been the best example, but basically it's about the fact that if you have a slot content, and the label prop, the label becomes the value of aria-label attribute

**Pull request merged**